src/lib.rs

```diff
@@ -1619,6 +1619,61 @@ impl<B: BitBlock> BitVec<B> {
     pub fn shrink_to_fit(&mut self) {
         self.storage.shrink_to_fit();
     }
+
+    /// Inserts a given bit at index `at`, shifting all bits after by one
+    ///
+    /// # Panics
+    /// Panics if `at > nbits`
```

Comment:
Could be more clear, saying Panics if the `at` argument is out of bounds for `BitVec`'s length.